content/raw/chat/identity_verification.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "identity verification spec"
+description: ""
+date: 2020-11-03T08:48:57+00:00
+lastmod: 2020-11-03T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Identity Verification
+
+## Motivation
+
+Users need a mechanism to verify a contact identity due to impersonation risks. For example someone can set their profile picture and username to be the same as a an individual the user knows. Here we describe one of the mechanisms used to deal with this which involve asking for verification of the identity of the user with a challenge question.
+
+## Designs
+
+- [sending identity verification request](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=670%3A339162)
+- [responding to an identity verification request](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1543%3A341560)
+
+## Use Cases
+
+### Ask to verify identity
+
+prequisite: user has been added as a contact.
+
+1. User selects a contact
+2. User prompts to verify identity
+3. Users writes a challenge question or request (for e.g something only the real person would know, or some action that only that person would be able to perform)
```

Comment:
If Status Desktop implements the sending of voice messages, wouldn't that be the most friction-less way to verify the identity of a contact? 
We could still combine it with a challenge/response but for most relationships just hearing the voice of your contact would be enough.